attrs/attrs.go

```diff
@@ -67,3 +67,16 @@ const (
 	Scope          = "scope"
 	Headers        = "headers"
 )
+
+// list of supported boolean attributes
+// see https://html.spec.whatwg.org/multipage/indices.html#attributes-3 for reference
+var BooleanAttrs = map[string]struct{}{
```

Comment:
- I'm not sure this needs to be publicly to people and since it's only used in `elem.go` can we move it there for now like the `voidElements`?
- Also, can we add the full list and add anything missing from the `attrs` constants to there. We can make separate ticket for that as well and link it in this as well or do it as a followup.

Reply:
> I'm not sure this needs to be publicly to people and since it's only used in elem.go can we move it there for now like the voidElements?

I think you are probably right, though I feel it's more proper to keep this boolean attrs list close to attrs themselves. But for sake of keeping things simple I agree that is completely fine to have it in `elem.go`.

> Also, can we add the full list and add anything missing from the attrs constants to there. We can make separate ticket for that as well and link it in this as well or do it as a followup.

I think it's better to add an attribute to the `booleanAttrs` map when we want't to support it in the code.